content/en/docs/faq/byop.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Bring your own Prometheus"
```

Comment:
Replace the built-in Prometheus with your own, how about this title?

Reply:
I agree

Reply:
BYO or bring your own is an idiom actually
![image](https://user-images.githubusercontent.com/18525465/99491922-9cb02980-29a7-11eb-8c82-2c45211b2b96.png)

Reply:
I see...I also noticed some expressions from Oracle and Azure. BYO is an idiom. Learned and thanks.